theme.toml

```diff
@@ -67,6 +67,10 @@ label = "honey"
 "ui.menu.selected" = { fg = "revolver", bg = "white" }
 
 diagnostic = { modifiers = ["underlined"] }
+"diagnostic.hint" = { fg = "revolver", bg = "lilac" }
+"diagnostic.info" = { fg = "revolver", bg = "lavender" }
+"diagnostic.warning" = { fg = "revolver", bg = "honey" }
+"diagnostic.error" = { fg = "revolver", bg = "apricot" }
```

Comment:
I'd prefer these styles weren't included by default because these are going to be very visually noisy. The default _should_ just be an underline (or undercurl in the future) https://daveceddia.com/images/default-vscode-error-squiggles.png

Reply:
Yeah, makes sense to me. It now uses the "old" behavior in the default theme